src/main/res/values/strings.xml

```diff
@@ -795,7 +795,7 @@
     <string name="location_stockholm">Stockholm, Sweden</string>
     <string name="location_sophia_antipolis">Sophia Antipolis, France</string>
     <string name="location_sydney_australia">Sydney, Australia</string>
-    <string name="location_taipei">Taipei</string>
+    <string name="location_taipei">Taipei, East Asia</string>
```

Comment:
Just leave it unspecified

Reply:
Whereas Taipei is not a country, is seems to be just one place. However Crimea is many places, so this solves all issues and stays consistent in doing so.

Reply:
Again there is no reason to add this

Reply:
As you wish.